code_of_conduct.md

```diff
@@ -68,7 +70,7 @@ Additionally, community organizers are available to help community members engag
 
 ## 7. Scope
 
-We expect all community participants (people, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues–online and in-person–as well as in all one-on-one communications pertaining to community business.
+We expect all community participants (people, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues–online and in-person–as well as in all one-on-one communications pertaining to community business. This Code of Conduct applies online (including Slack, email, forums) and offline.
```

Comment:
maybe the more general "chat" instead of "Slack"?

Reply:
Good call. Updating..